.vscode/settings.json

```diff
@@ -25,9 +25,9 @@
     "eslint.enable": false,
     "cSpell.allowCompoundWords": true,
     "cSpell.caseSensitive": true,
-    "rust-analyzer.checkOnSave.command": "clippy",
+    // "rust-analyzer.checkOnSave.command": "clippy",
```

Comment:
Running `clippy`  when onSave is slow on my computer, I will recover it when this pr is finished.

Reply:
Can you revert this?

Reply:
Done